test/e2e/tokens_test.go

```diff
@@ -50,6 +50,7 @@ func (suite *TokensTestSuite) TestE2EFungibleTokensAsync() {
 	}
 	CreateTokenPool(suite.T(), suite.testState.client1, pool, false)
 
+	<-received1
 	<-received1
```

Comment:
Could you add comments to notate why we get double events? (one is the pool creation, one is the pool announcement)